<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCoordinateForm</class>
 <widget class="QFrame" name="CCoordinateForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QGridLayout" name="gl_CoordinateSelector">
   <item row="1" column="2">
    <widget class="swift::gui::CTickLabel" name="lblp_LatCheck">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="swift::gui::CTickLabel" name="lblp_ElvCheck">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="8">
    <widget class="QLineEdit" name="le_LngSecFrag">
     <property name="placeholderText">
      <string>fract.sec</string>
     </property>
    </widget>
   </item>
   <item row="5" column="5" colspan="4">
    <widget class="QFrame" name="fr_Buttons">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="hl_Buttons">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="hs_Buttons">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pb_OwnAircraft">
        <property name="text">
         <string>own aircraft</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_Set">
        <property name="text">
         <string>set</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="5" colspan="4">
    <widget class="QLineEdit" name="le_Location">
     <property name="placeholderText">
      <string>ICAO, city</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QLineEdit" name="le_Z">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QLineEdit" name="le_LngSec">
     <property name="placeholderText">
      <string>sec</string>
     </property>
    </widget>
   </item>
   <item row="3" column="7">
    <widget class="QLineEdit" name="le_X">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QLineEdit" name="le_LngMin">
     <property name="placeholderText">
      <string>min</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lbl_Elevation">
     <property name="text">
      <string>Elevation:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QLineEdit" name="le_LngDeg">
     <property name="placeholderText">
      <string>deg.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QRadioButton" name="rb_S">
     <property name="text">
      <string>S</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">bg_NS</string>
     </attribute>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QRadioButton" name="rb_E">
     <property name="text">
      <string>E</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">bg_EW</string>
     </attribute>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbl_Latitude">
     <property name="text">
      <string>Latitude (0°-90°):</string>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QLineEdit" name="le_LatSecFrag">
     <property name="placeholderText">
      <string>fract.sec</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QRadioButton" name="rb_N">
     <property name="text">
      <string>N</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">bg_NS</string>
     </attribute>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QRadioButton" name="rb_W">
     <property name="text">
      <string>W</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">bg_EW</string>
     </attribute>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QLabel" name="lbl_Sec">
     <property name="text">
      <string>sec</string>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QLabel" name="lbl_FracSec">
     <property name="text">
      <string>fract.sec</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="le_Latitude">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>WGS 0° 0' 0.000&quot; N/S</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="lbl_Min">
     <property name="text">
      <string>min</string>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QLineEdit" name="le_Y">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLineEdit" name="le_LatMin">
     <property name="placeholderText">
      <string>min</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lbl_Longitude">
     <property name="text">
      <string>Longitude (0°-180°):</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="le_Longitude">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>WGS 0° 0' 0.000&quot; E/W</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="2">
    <widget class="QLabel" name="lbl_Location">
     <property name="text">
      <string>Location:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLineEdit" name="le_LatDeg">
     <property name="placeholderText">
      <string>deg.</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>X/Y/Z:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="le_Elevation">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QLineEdit" name="le_LatSec">
     <property name="placeholderText">
      <string>sec</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="swift::gui::CTickLabel" name="lblp_LngCheck">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="lbl_Deg">
     <property name="text">
      <string>deg</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::CTickLabel</class>
   <extends>QLabel</extends>
   <header>gui/ticklabel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>le_Latitude</tabstop>
  <tabstop>rb_N</tabstop>
  <tabstop>rb_S</tabstop>
  <tabstop>le_LatDeg</tabstop>
  <tabstop>le_LatMin</tabstop>
  <tabstop>le_LatSec</tabstop>
  <tabstop>le_LatSecFrag</tabstop>
  <tabstop>le_Longitude</tabstop>
  <tabstop>rb_E</tabstop>
  <tabstop>rb_W</tabstop>
  <tabstop>le_LngDeg</tabstop>
  <tabstop>le_LngMin</tabstop>
  <tabstop>le_LngSec</tabstop>
  <tabstop>le_LngSecFrag</tabstop>
  <tabstop>le_Elevation</tabstop>
  <tabstop>le_Location</tabstop>
  <tabstop>pb_OwnAircraft</tabstop>
  <tabstop>pb_Set</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="bg_NS"/>
  <buttongroup name="bg_EW"/>
 </buttongroups>
</ui>
